stackstorm/handler.py

```diff
@@ -93,7 +104,32 @@ def _load_config_schemas():
 CONFIG_SCHEMAS = _load_config_schemas()
 
 
-def base(event, context, passthrough=False):
+def base(event, context, passthrough=False, debug=False):
+    # Set up logging
+    # TODO: Allow log level to be specified by the user, default to DEBUG when
+    # --verbose is used
+    logger = logging.getLogger()
+
+    if debug:
+        logger.setLevel(logging.DEBUG)
+    else:
+        logger.setLevel(logging.INFO)
+
+    try:
+        if isinstance(event, basestring):
+            event = json.loads(event)
+    except ValueError as e:
+        print("ERROR: Can not parse `event`: '{}'\n{}".format(str(event), str(e)))
```

Comment:
We can go ahead replace print with `LOG.*`, right?

Reply:
Yeah, either works. 

print also ends up in the log but without the log level, etc. prefix so LOG might indeed be better, yeah.